### ISSUE TITLE
Elpy Anaconda sudden problem

### ISSUE THREAD

**Issue by DrWaleedAYousef** [jorgenschaefer/elpy#1542]
All of a sudden I started getting this message when I try to run any python code:
`version-to-list: Invalid version syntax: ‘3.6.8 :: Anaconda custom (64-bit)’`

Thank you for reporting to Elpy !

Here is a template to help you do it so your issue can be fixed as
soon as possible. Please replace this paragraph with a summary of
your issue and fill the next sections.

# Steps to reproduce

# My configuration
## OS
Archlinux

## Result of `(elpy-config)`
```
Elpy Configuration

Virtualenv........: MyDefaultEnv (/home/wyousef/Downloads/AAProgramsAA/anaconda3/envs/MyDefaultEnv)
RPC Python........: 3.6.8 (/home/wyousef/Downloads/AAProgramsAA/anaconda3/envs/MyDefaultEnv/bin/python)
Interactive Python: python (/home/wyousef/Downloads/AAProgramsAA/anaconda3/envs/MyDefaultEnv/bin/python)
Emacs.............: 26.1
Elpy..............: 1.28.0
Jedi..............: 0.13.2
Rope..............: 0.11.0
Autopep8..........: Not found (1.4.3 available)
Yapf..............: Not found (0.26.0 available)
Black.............: Not found (18.9b0 available)
Syntax checker....: Not found (flake8)

The autopep8 package is not available. Commands using this will not work.

```

## Elpy configuration in my init.el
```
(use-package elpy
  :ensure t
  :init
  (elpy-enable)
  :config
  (setenv "WORKON_HOME" "~/Downloads/AAProgramsAA/anaconda3/envs/")
  (pyvenv-mode 1)
  (pyvenv-workon "MyDefaultEnv")
  (setq elpy-shell-use-project-root nil)
  )

```

**Comment by galaunay** [comment]
Thanks for reporting, 

That is coming for a recent commit.
It should be fixed with the merging of #1543.

Please tell me if it is not.

**Comment by DrWaleedAYousef** [comment]
Thanks so much. It is done!
